### PR TITLE
fixes #4

### LINKED ISSUE #4: Developer tooling for Voice Compass SDK
When user creates a voice compass client on the frontend, in addition to `botId`, `apiKey` and `journeyId`, they should also be able to add a `debug` optional boolean flag. When this flag is active, there should be nice logs in the console whenever a step is called, with all related metadata/payload/journey info. This can be simple at first - since we need this sooner rather than later - but eventually we can make it nice the way fancy redux loggers do it.

### PR DESCRIPTION
**Pull request by chulander**
simple PR to handle emoji debug flag logging

### CODE REVIEW

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -93,17 +94,33 @@ export const create = (config: Config): VoiceCompass => {
...
+      ...stepData,
+      contactId: config.contactId,
+      botId: config.botId,
+      journeyId: stepData.journeyId || config.journeyId,
+      voice: config.voice,
+      language: config.language,
+    };
+
+    if (config.debug) {
+      console.info(
```
Comment (peterszerzo) [inline packages/core/src/index.ts]: Move this inside the `then` block so it fires exactly as the request completes.

File: packages/core/src/index.ts [hunk 2]
```diff
@@ -93,17 +94,33 @@ export const create = (config: Config): VoiceCompass => {
...
+        `${String.fromCodePoint(0x02713)}: step: ${_payload.stepId}`,
```
Comment (peterszerzo) [inline packages/core/src/index.ts]: Can we drop the first of the two `:`'s?

**Review by peterszerzo — CHANGES_REQUESTED** [review]

File: packages/core/src/index.ts [hunk 3]
```diff
@@ -93,17 +94,33 @@ export const create = (config: Config): VoiceCompass => {
       return;
     }
 
+    const _payload = {
```
Comment (peterszerzo) [inline packages/core/src/index.ts]: What do you mean to imply with the `_` prefix in the naming? Can we think of a way to make the name indicate this intention explicitly? These underscores have lost quite a bit of semantic meaning for me, maybe you see this differently?

Comment (chulander) [reply packages/core/src/index.ts]: how about payload?

Comment (chulander) [reply packages/core/src/index.ts]: i'm misusing the underscore convention, it was a way to indicate private / non-exported variables which doesn't even make sense thinking about it bc its being passed into an http call. soo yeah. i wont use it going forward and will think of a better name

Comment (peterszerzo) [reply packages/core/src/index.ts]: `payload` makes sense to me :)

**Review by peterszerzo — APPROVED** [review]

**Pull request merged**